tests/test_visitors/test_ast/test_complexity/test_overuses/test_overused_expressions.py

```diff
@@ -170,3 +174,52 @@ def test_class_expression_use(
     visitor.run()
 
     assert_errors(visitor, [])
+
+
+@pytest.mark.parametrize('code', [
+    function_context1,
+    function_context2,
+    function_context3,
+    function_context4,
+    method_context1,
+    method_context2,
+    method_context3,
+])
+@pytest.mark.parametrize('expression', ignored_expressions)
+def test_func_ignored_expressions(
+    assert_errors,
+    parse_ast_tree,
+    options,
+    expression,
+    code,
+    mode,
+):
+    """Ensures that ignored expressions does not raise violations."""
+    tree = parse_ast_tree(mode(code.format(expression, expression)))
+
+    option_values = options(max_function_expressions=1)
+    visitor = ExpressionOveruseVisitor(option_values, tree=tree)
+    visitor.run()
+
+    assert_errors(visitor, [])
+
+
+@pytest.mark.parametrize('code', [
+    module_context,
```

Comment:
Why this is a separate case? Why not using `test_func_ignored_expressions`?

Reply:
Ah, I just copied the tests from above and they had different functions for the two cases. Digging through the code, I now understand that it is unnecessary. Will fix it.